packages/editor/src/components/local-autosave-monitor/index.js

```diff
@@ -88,45 +62,74 @@ function useAutosaveNotice() {
 	const {
 		postId,
 		getEditedPostAttribute,
-	} = useSelect( ( select ) => ( {
-		postId: select( 'core/editor' ).getCurrentPostId(),
-		getEditedPostAttribute: select( 'core/editor' ).getEditedPostAttribute,
-	} ) );
+		remoteAutosave,
+		hasFetchedAutosave,
+	} = useSelect( ( select ) => {
+		const _postId = select( 'core/editor' ).getCurrentPostId();
```

Comment:
Nit: I'd just drop the `_`  as we don't really use that pattern elsewhere.

Reply:
I guess it's because  it's already use up, I  wonder if we can rename it something else? otherwise, it's fine.

Reply:
Yeah, it's to avoid collision. I think I've seen the `_` usage somewhere else recently. Happy to change to whatever is best.